docs/integrations/gerrit/locations.md

```diff
@@ -0,0 +1,36 @@
+---
+id: locations
+title: Gerrit Locations
+sidebar_label: Locations
+description: Integrating source code stored in Gerrit into the Backstage catalog
+---
+
+The Gerrit integration supports loading catalog entities from Gerrit hosted gits. Entities can
+be added to [static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+To use this integration, add configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  gerrit:
+    - host: gerrit.company.com
+      apiBaseUrl: gerrit.company.com/gerrit
+      userName: ${GERRIT_USERNAME}
```

Comment:
seems like a typo: `userName` vs `username`. At other places, you use `username`

Reply:
"username" it is!